source/changelog.txt

```diff
@@ -24,6 +24,8 @@ v2.0.0
   - ``db.enableFreeMonitoring``
   - ``db.disableFreeMonitoring``
 
```

Comment:
(suggestion)
I recommend adding a changelog section called Compatibility Considerations. Or Operating System Compatibility or similar. This makes the 2.0.0 change logs a little more scannable.

Reply:
Will do.